### DISCUSSION TITLE
2D Gravity Forward Modeling

### DISCUSSION THREAD

**Discussion by diogoocruz** [fatiando/community#130] [Q&A]
Hello!
I'm new to geophysics and Fatiando. 
I saw that there is Choclo to do 3d Forward modeling. However, I would like to know if there is something like Moulder from the legacy Fatiando library in the "new" Fatiando. I would like to simulate the gravimetric anomaly caused by a 2d model (profile). 
Thank you in advance!

**Comment by leouieda** [comment]
Hi @diogoocruz, welcome to Fatiando!

Several years ago we started a repository that would be an updated Moulder: https://github.com/fatiando/moulder But as you can see from the repository, we never found the time to dedicate to it and actually update the code. What is there is mostly a copy of what was in the v0.5 of `fatiando`. It needs some work to be useful but it's a good place to start. 

If you're interested in getting involved in the project, this could be a great place to start! Let us know and we can help you get started. A good place to start is by asking in our chat and showing up to one of our weekly calls: https://www.fatiando.org/contact/

Feel free to use/adapt/update the code from that repository. I'm hoping to find some time next year to dedicate to this since I want to use it in my classes again but that may not happen.

**Reply by diogoocruz** [reply]
Hello!
I've just started looking at it! I'm having some problems installing the old "fatiando".  However, I'd like to try to help and contribute :)
I'm also getting this error when I try to log into the Mattermost channel:

![image](https://github.com/fatiando/community/assets/92991667/fede2150-a6c4-4087-8614-bf9ddf481311)

**Reply by santisoler** [reply]
Thanks @diogoocruz for letting us know. Swung changed the name of the Team therefore the link stopped working. I'll fix it!

**Reply by MGomezN** [reply]
Hello @diogoocruz! I'm using this link to access Mattermost https://mattermost.softwareunderground.org/
Hopefully you can join us soon :)
